app/process/handler/message/handler.go

```diff
@@ -64,6 +64,8 @@ func NewHandler(
 			Name: constants.ValidatorsParticipationRateGaugeName,
 			Help: constants.ValidatorsParticipationRateGaugeHelp,
 		})
+		// Set to 100 in order not to fire alerts until a transaction is processed
+		participationRate.Set(100)
```

Comment:
Maybe we can have the number as a constant?

Reply:
yep good idea